practice-two/src/js/services/httpClient.service.js

```diff
@@ -0,0 +1,69 @@
+import axios from 'axios';
+import { ERROR_MESSAGES } from '../constants/messages';
+
```

Comment:
Let try to define a HttpService that includes get, post, put, delete. It just takes endpoint as a constructor argument so that you don't need to pass the endpoint everytime you use

```
const API_BASE_URL = process.env.API_BASE_URL
class HttpService {
  constructor(baseUrl) {
    this.endpoint = endpoint;
    this.axiosClient = axios.create({
      headers: {
        accept: 'application/json',
      },
    });
  }

get(){
   return this.axiosClient.get(`${API_BASE_URL}${endpoint}`);
}
```